dotCMS/src/main/java/com/dotcms/util/ReflectionUtils.java

```diff
@@ -160,4 +160,22 @@ public static final Object newInstance (final String className) {
         return clazz;
     }
 
+    /**
+     * Java Beans set value utility
+     * The method d
+     * @param target any object
+     * @param method any method that takes one single param it doesnt have to be named setSomething
+     * @param value The param expected by the method we intent to fire
+     */
+    public static void setValue(final Object target, final String method,final Object value) {
```

Comment:
good, do you have a test?

Reply:
nope but the value being set is covered by the other test.